changelog.md

```diff
@@ -1,3 +1,32 @@
+## 3.5.0 / 2021-xx-xx
+
+### New Features
+
+- All users can now set a public user profil
+
+### Fixed Issues
+
+- Frontend: [#3578](https://github.com/tracim/tracim/issues/3578),
+[#3785](https://github.com/tracim/tracim/issues/3785),
+[#3790](https://github.com/tracim/tracim/issues/3790),
+[#3989](https://github.com/tracim/tracim/issues/3989),
+[#3999](https://github.com/tracim/tracim/issues/3999)
+- Other: [#4062](https://github.com/tracim/tracim/issues/4062)
+
+### Known Issues
+
+- SQLite: IDs of objects can be re-used on database (issue [#4016](https://github.com/tracim/tracim/issues/4016))
+
+### Breaking Changes
+
+- Some of renaming and wording changes in frontend (issue [#3901](https://github.com/tracim/tracim/issues/3901))
```

Comment:
```suggestion
- Wording changes have been done in the frontend (issue [#3901](https://github.com/tracim/tracim/issues/3901))
```

Reply:
I use the proposal "Some wording in the user interface are changed" because its more clear to use user interface in place of frontend.